lib/generators.nix

```diff
@@ -559,44 +740,55 @@ ${expr "" v}
     else
       abort "generators.toLua: type ${typeOf v} is unsupported";
 
-  /*
-   Mark string as Lua expression to be inlined when processed by toLua.
+  /**
+    Mark string as Lua expression to be inlined when processed by toLua.
+
+
+    # Inputs
+
+    `expr`
+
+    : 1\. Function argument
 
-   Type:
-     mkLuaInline :: String -> AttrSet
+    # Type
+
+    ```
+    mkLuaInline :: String -> AttrSet
+    ```
   */
   mkLuaInline = expr: { _type = "lua-inline"; inherit expr; };
+} // {
+  /**
+    Generates JSON from an arbitrary (non-function) value.
+    For more information see the documentation of the builtin.
 
-in
-
-# Everything in this attrset is the public interface of the file.
-{
-  inherit
```

Comment:
Inheriting the names here breaks the source position tracking of `unsafeGetAttrPos`. Although nixdoc may support getting the correct comments. All source positions will link to the same position.